packages/api/src/services/company.service.ts

```diff
@@ -0,0 +1,143 @@
+import { companies, CompaniesSchema, companies_users, users } from "../schemas";
+import { db } from "../db";
+import { AppError } from "../utils/ExpressError";
+import { normalizeUser } from "./user.service";
+import { and, eq } from "drizzle-orm";
+import { number } from "zod";
+
+type Company = {
+  id: number;
+  createdAt: Date;
+  updatedAt: Date;
+  name: string;
+  logo: string | null;
+  blurb: string | null;
+};
+export enum UserCompanyRole {
+  Owner = 0,
+  Admin = 10,
+  Editor = 20,
+  Contributor = 30,
+  deleted = 1000,
```

Comment:
Not a fan of this personally, but if you choose to do it, then use a capital:

```suggestion
  Deleted = 1000,
```